### PR TITLE
chore(LICENSE): add Apache Software 2.0

### LINKED ISSUE #38: Missing license

### PR DESCRIPTION
**Pull request by tbobm**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (nicolasdecorbez) [inline LICENSE:189]: Is this line mean to be replaced ? like : 
```suggestion
   Copyright 2022 Theo "Bob" Massard
```

**Review by nicolasdecorbez — COMMENTED** [review]
now better see how to license, thanks mate ! 

small suggestion on line 189 tho

**Comment by nicolasdecorbez** [comment]
Closes #38 btw.

File: LICENSE (hunk 1 above)
Comment (tbobm) [reply LICENSE:189]: nice catch lmao

**Pull request merged**